tests/backends/tests.py

```diff
@@ -1211,3 +1211,16 @@ def create_object():
         thread.join()
 
         self.assertEqual(models.Object.objects.count(), 2)
+
+
+@override_settings(DEBUG=True)
```

Comment:
How about something less repetitive like:
```python
    def test_auto_is_null_auto_config(self):
        query = "set sql_auto_is_null = 0"
        connection.init_connection_state()
        last_query = connection.queries[-1]['sql'].lower()
        if connection.features.can_return_last_inserted_id_with_auto_is_null:
            self.assertIn(query, last_query)
        else:
            self.assertNotIn(query, last_query)
```
Also, could you create a separate `test_mysql.py` file for the new test?